discord/ui/view.py

```diff
@@ -467,6 +467,20 @@ async def wait(self) -> bool:
         """
         return await self.__stopped
 
+    def disable_all_items(self, *, exclusions: Optional[List[int]]) -> None:
+        """
+        Disables all items in the view.
+
+        Parameters
+        ----------- 
+
+        exclusions: 
+            Optional[List[:class:`int`]] A list of item indexes in `self.children` to not disable from the view.
```

Comment:
```suggestion
        exclusions: Optional[List[:class:`int`]]
            A list of item indexes in `self.children` to not disable from the view.
```

Reply:
> I think it would make sense to also support lists of `Item` objects into the exclusions param.
-> Yes, then it shall completely be the `child` and no the indexes.